packages/mainsail/source/transaction.service.ts

```diff
@@ -35,6 +39,37 @@ import { network } from "./crypto/networks/devnet/network.js";
 import { MultiSignatureSigner } from "./multi-signature.signer.js";
 import { Request } from "./request.js";
 
+export const getApp = async () => {
```

Comment:
We will eventually create a specific solution to have an app instance, this is temporary.

Reply:
Done